src/odemis/gui/cont/tabs.py

```diff
@@ -499,6 +501,25 @@ def __init__(self, name, button, panel, main_frame, main_data):
                 self
             )
 
+    def _on_overview_visible(self, val):
+        """
+        Apply the visibility status of overview streams in the Localisation tab to
+        the overview streams in the Chamber tab.
+        """
+        # All the overview streams
+        acquired_streams = set()
+        acquired_streams.update(self.tab_data_model.overviewStreams.value)
```

Comment:
Let's call them `ov_streams`, as acquired is a misnomer (because there are other acquired streams which are not in this list).
Also you can simply write:
`ov_streams = set(self.tab_data_model.overviewStreams.value)`